README.md

```diff
@@ -142,11 +142,11 @@ For more information on how and why we built this dashboard, check out this [blo
 
 ## Discussions
 
-[GitHub Discussions](https://docs.github.com/discussions/quickstart) is a feature to enable fluid and open discussions within the community
+[GitHub Discussions](https://docs.github.com/discussions/quickstart) is a feature to enable free and open discussions within the community
 for topics that are not related to code, unlike issues.
 
-This is an experiment we are trying in our repositories to see if it helps move discussions out of issues so that issues remain actionable by the team or members of the community.
-There should be no expectation that PowerShell team members are regular participants in the discussions.
+This is an experiment , we are trying in our repositories to see if it helps move discussions out of issues so that issues remain actionable by the team or members of the community.
```

Comment:
```suggestion
This is an experiment we are trying in our repositories, to see if it helps move discussions out of issues so that issues remain actionable by the team or members of the community.
```